src/core_plugins/kibana/server/tutorials/apm/apm_server_instructions.js

```diff
@@ -0,0 +1,63 @@
+const EDIT_CONFIG = {
+  title: 'Edit the configuration',
+  textPre: `If you're using an X-Pack secured version of Elastic Stack, you need to specify` +
+    `credentials in the apm-server.yml config file.`,
+  commands: [
+    'output.elasticsearch:',
+    '    hosts: ["<es_url>"]',
+    '    username: <username>',
+    '    password: <password>'
+  ]
+};
+
+const IMPORT_DASHBOARD = {
+  title: 'Import dashboards',
+  textPre: 'APM Server ships with pre-configured dashboards'
+};
+
+const START_SERVER = {
+  title: 'Start APM Server',
+  textPre: 'The server processes and stores application performance metrics in Elasticsearch'
+};
+
+export const DOWNLOAD_SERVER = {
+  title: 'Download and unpack APM Server',
+};
+
+export const UNIX_FAMILY_SERVER_INSTRUCTIONS = [
+  {
+    ...IMPORT_DASHBOARD,
+    commands: [
+      './apm-server setup'
+    ]
+  },
+  EDIT_CONFIG,
+  {
+    ...START_SERVER,
+    commands: [
+      './apm-server -e'
+    ]
+  }
+];
+
+export const WINDOWS_SERVER_INSTRUCTIONS = [
+  {
+    ...DOWNLOAD_SERVER,
+    commands: [
+      'TODO'
```

Comment:
@nreese Where can I see the Metricbeat download instructions to compare? I was looking here https://github.com/elastic/kibana/tree/master/src/core_plugins/kibana/server/tutorials

Reply:
https://github.com/elastic/kibana/blob/master/src/core_plugins/kibana/common/tutorials/metricbeat_instructions.js